proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -165,6 +165,28 @@ message OrderFillEventV1 {
   uint64 total_filled_taker = 8;
 }
 
+// DeleveragingEvent message contains all the information for a deleveraging
+// on the dYdX chain. This includes the liquidated/offsetting subaccounts and
+// the amount filled.
+message DeleveragingEventV1 {
+  // ID of the subaccount that was liquidated.
+  dydxprotocol.indexer.protocol.v1.IndexerSubaccountId liquidated = 1
+      [ (gogoproto.nullable) = false ];
+  // ID of the subaccount that was used to offset the position.
+  dydxprotocol.indexer.protocol.v1.IndexerSubaccountId offsetting = 2
+      [ (gogoproto.nullable) = false ];
+  // The ID of the clob pair that was liquidated.
+  uint32 clob_pair_id = 3;
```

Comment:
nit: let's maybe use `perpetual_id` instead of clob_pair_id here.

CLOB actually assumes a one to many relationship between perp id and clob pair id ([link](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/x/clob/keeper/keeper.go#L34)). We currently only have 1 to 1 mapping, but using perpetual id is probably more resilient to changes in the future

Reply:
for liquidations we send both. I'm going to include both.